components/admin_console/system_users/system_users.tsx

```diff
@@ -354,6 +354,7 @@ export default class SystemUsers extends React.PureComponent<Props, State> {
                         <option value={UserFilters.SYSTEM_ADMIN}>{Utils.localizeMessage('admin.system_users.system_admin', 'System Admin')}</option>
                         <option value={UserFilters.SYSTEM_GUEST}>{Utils.localizeMessage('admin.system_users.guest', 'Guest')}</option>
                         <option value={UserFilters.INACTIVE}>{Utils.localizeMessage('admin.system_users.inactive', 'Inactive')}</option>
+                        <option value={UserFilters.ACTIVE}>{Utils.localizeMessage('admin.system_users.active', 'Active')}</option>
```

Comment:
nit : Can we move active before inactive?

Reply:
done